17:51 < aurel32> _rene_: for libreoffice on mips, either you need an old 
                 gcc-4.7 (e.g. the one from wheezy), or a binutils matching the 
                 changes from gcc-4.7 (e.g the one from experimental)
17:51 < aurel32> well s/gcc-4.7/gcc-4.6/
17:53 < _rene_> well, I have a workaround now (which is building), so...
17:53 < _rene_> but thanks for the info
17:53 < _rene_> aurel32: I do exactly as gcc says and link with that lib. seems 
                to work
17:53 < _rene_> aurel32: (that's what the LO build on gabrielli currently does)
17:54 < aurel32> at some point we'll need a binutils upload to fix that properly
[...]
18:00 < _rene_> aurel32: ok, to summarize: both gcc-4.6 and -4.7 sid don't 
                "work". only with experimentals binfilter?
18:01 < _rene_> aurel32: /me could then remove the patch and add a appropriate 
                build-dep when the new binutils is in sid..
18:03  * _rene_ can try with that build-dep with LO 4.1 beta1 (to be released 
          and maybe uploaded next week) in experimental :)
18:08 -!- zwenna [~user@p4FC60FB4.dip0.t-ipconnect.de] has joined 
          #debian-release
18:08 < aurel32> _rene_: as far as i know yes, gcc-4.6 and -4.7 from sid (and 
                 soon from testing) need the experimental binutils
18:09 < aurel32> _rene_: 
                 http://www.sourceware.org/ml/binutils/2012-08/msg00064.html
18:10 < _rene_> ok. /me will add a comment to the patch and adapt the changelog 
                then.

diff -urN cpputools.old/Executable_sp2bv.mk cpputools/Executable_sp2bv.mk
--- a/cpputools.old/Executable_sp2bv.mk	2013-04-30 14:54:23.000000000 +0200
+++ b/cpputools/Executable_sp2bv.mk	2013-05-17 21:41:33.373549555 +0200
@@ -34,6 +34,7 @@
     cppu \
     cppuhelper \
     sal \
+    xmlreader \
 ))
 
 $(eval $(call gb_Executable_add_exception_objects,sp2bv,\
